ccmlib/cluster.py

```diff
@@ -400,15 +403,15 @@ def get_seeds(self):
 
     def show(self, verbose):
         msg = "Cluster: '%s'" % self.name
-        print_(msg)
-        print_('-' * len(msg))
+        logger.debug(msg)
```

Comment:
in this function please revert the logger it need to print to stdout directly

Reply:
all messages are print to the stdout